djangocms_versioning/templates/djangocms_versioning/admin/archive_confirmation.html

```diff
@@ -9,8 +9,7 @@
 {% endblock %}
 
 {% block breadcrumbs %}{% endblock %}
-{% block bodyclass %}{{ block.super }} app-{{ opts.app_label }} model-{{ opts.model_name }}
-delete-confirmation{% endblock %}
+{% block bodyclass %}{{ block.super }} app-{{ opts.app_label }} model-{{ opts.model_name }}delete-confirmation{% endblock %}
```

Comment:
This is missing a space before `delete-confirmation` now

Reply:
weird!! There is space in my local code.